omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -19,7 +19,6 @@
   'agent_metrics',
   'docker_daemon',
   'kubernetes',
-  'kubernetes_state',
```

Comment:
I can't remember, has this been deprecated?

Reply:
it seems alive and kickin' https://github.com/DataDog/integrations-core/tree/master/kubernetes_state but I'm not sure whether we're going to kill it or what

Reply:
indeed, 6.0 needs to ship with it

as the ksm payloads get bigger and bigger (12k metrics in some clusters), we might investigate reworking it, but that's 2018 material